testing/main.tf

```diff
@@ -0,0 +1,111 @@
+provider "aws" {
+  region                      = var.aws_region
+  skip_credentials_validation = true
+  skip_requesting_account_id  = true
+}
+
+###
+### SECURITY GROUPS
+###
+resource "aws_security_group" "allow-all" {
+  name   = "${var.tf_user}-allow-all-${var.GITHUB_RUN_ID}"
+  vpc_id = "vpc-07402b459d3b18976"
+
+  ingress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = {
+    Name       = "allow-all"
+    Owner      = var.tf_user
+    github_run = "${var.GITHUB_RUN_ID}"
+  }
+}
+
+###
+### EC2 INSTANCES
+###
+# EC2 Instance for RKE2 Cluster - Manager
+resource "aws_instance" "control_node" {
+  count = var.control_nodes
+
+  ami           = lookup(var.amis[var.aws_region], var.os)
+  instance_type = var.instance_type
+  subnet_id     = "subnet-0523d8467cf8e5cec"
+  key_name      = "rke2-ansible-ci"
+  root_block_device {
+    volume_type = "standard"
+    volume_size = 30
+  }
+
+  vpc_security_group_ids = [aws_security_group.allow-all.id]
+
+  tags = {
+    Name        = "rke2_ansible-testing-server-${var.os}-${var.GITHUB_RUN_ID}-${count.index}"
+    StopAtNight = "True"
+    Owner       = var.tf_user
+    NodeType    = "Server"
+    github_run  = "${var.GITHUB_RUN_ID}"
+  }
+
+  provisioner "remote-exec" {
+    connection {
+      host        = coalesce(self.public_ip, self.private_ip)
+      type        = "ssh"
+      user        = "ubuntu"
+      private_key = file(pathexpand(".key"))
+    }
+    inline = [
+      "uptime",
+    ]
+  }
+}
+
+# EC2 Instance for RKE2 Cluster - Workers
+resource "aws_instance" "worker_node" {
+  count = var.worker_nodes
+
+  ami                         = lookup(var.amis[var.aws_region], var.os)
+  instance_type               = var.instance_type
+  subnet_id                   = "subnet-0523d8467cf8e5cec"
```

Comment:
same as line 45, var here might be better

Reply:
Fixing.